ecl/eclagent/eclagent.cpp

```diff
@@ -3129,7 +3129,7 @@ char * EclAgent::getGroupName()
 {
 #ifdef _CONTAINERIZED
     // in a containerized setup, the group is moving..
-    return strdup("unknown");
+    return strdup("");
```

Comment:
let's revise the comment here now the interpretation has changed.
Replace the comment above with:

// In containerized there is no associated cluster group.
// Return an empty result instead, which if used for destinationGroup
// via Thorlib.group() as part of a fileservices call, will be interpreted
// as targeting the default data plane